test/functions/diten/formalArrayWithQueriedEltSize.future

```diff
@@ -1,3 +1,3 @@
-bug: array formal with queried element size trips internal error
+feature request: ability to query aspects of an array formal's elements
```

Comment:
Should this be an "unimplemented feature" ?

Reply:
It feels on the cusp to me, but slightly more on the feature request side of things.  I don't believe we ever intended to support this pattern in the original language design or specification; but it seems like a reasonable extension to what we currently have.